config/awesome/floppy/widget/mpd/mpd-music-updater.lua

```diff
@@ -15,7 +15,7 @@ local apps = require('configuration.apps')
 
 local update_cover = function()
 	
-	local extract_script = [[
+	local extract_script = [=[
```

Comment:
Why is there an equal there? Is it a typo?

Reply:
It's not a typo, that's how you can use `]]` for the bash if conditional inside the lua multistring. You can have any number of `=` as long as they are equal numbered.

Reply:
Oh, I see. Thanks for the info! Never knew that before.

Reply:
You can check it [here](http://lua-users.org/wiki/StringsTutorial) at **Nesting quotes** (lua's weird ikr)

Reply:
Thanks! Yeah lua's weird. Anyway, thanks for this. I'll just apply the patch to the other setups.

Reply:
Always glad to help :) Love my WM thanks to you, least I can do.

Reply:
Nice! Also, thanks for the nesting quotes. It's time to clean those bash scripts inside lua lol